packages/commerce-sdk-react/src/hooks/useCommerceApi.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+    ShopperBaskets,
+    ShopperContexts,
+    ShopperCustomers,
+    ShopperLogin,
+    ShopperOrders,
+    ShopperProducts,
+    ShopperPromotions,
+    ShopperDiscoverySearch,
+    ShopperGiftCertificates,
+    ShopperSearch
+} from 'commerce-sdk-isomorphic'
+
+import {ApiClients} from './types'
+
+const config = {
+    proxy: 'https://localhost:3000',
+    headers: {},
+    parameters: {
+        clientId: '<your-client-id>',
+        organizationId: '<your-org-id>',
+        shortCode: '<your-short-code>',
+        siteId: '<your-site-id>'
+    },
+    throwOnBadResponse: true
+}
+
+const useCommerceApi = (): ApiClients => {
```

Comment:
Let's be consistent with the letter-casing of "api". In the Provider component, it's all capitalized. But in this file, it's not.